src/BTCPayServer.Lightning.Common/PayResponse.cs

```diff
@@ -7,17 +7,18 @@ namespace BTCPayServer.Lightning
     public enum PayResult
     {
         Ok,
-        CouldNotFindRoute
+        CouldNotFindRoute,
+        Error
     }
     public class PayResponse
     {
-        public PayResponse(PayResult result)
+        public PayResponse(PayResult result, string errorDetail = null)
```

Comment:
in the future, please prefer adding an overload rather than a null default. This has this kind of side effect: https://circleci.com/gh/btcpayserver/btcpayserver/7359

Reply:
I should've re-linked project dependencies and tried out the usage of lib. Got lost into looking for updating NuGet version, only later realizing we need to move those NuGet powershell scripts to CircleCi.